package.json

```diff
@@ -33,7 +33,8 @@
   },
   "resolutions": {
     "@types/react": "^18",
-    "@types/react-dom": "^18"
+    "@types/react-dom": "^18",
+    "lru-cache": "^11.1.0"
```

Comment:
This fixed the failures, but is that the right way to resolve this? 🤔

Reply:
Agreed this is fine for now, need to make sure to check in and see if it can be removed down the line.